build/rpms/BUILD

```diff
@@ -0,0 +1,45 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@bazel_tools//tools/build_defs/pkg:rpm.bzl", "pkg_rpm")
+
+pkg_rpm(
+    name = "kubectl",
+    changelog = "CHANGELOG",
+    data = [
+        "//cmd/kubectl",
+    ],
+    spec_file = "kubectl.spec",
+    version = "1.6.0",
```

Comment:
this doesn't seem right.

do we need to get some sort of stamping support in `pkg_rpm`?